servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -256,6 +256,10 @@ def cromwell_query_params(query, page, page_size):
     if query.statuses:
         statuses = [{'status': s} for s in set(query.statuses)]
         query_params.extend(statuses)
+    if query.labels:
+        labels = [{'label': k + ':' + v} for k, v in query.labels.items()]
```

Comment:
I'm not sure if it will change in the future, but currently, Cromwell doesn't allow non-string key/value for labels, so I didn't add `str(k)` and `str(v)` here.